.devcontainer/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM kong/kong:2.7.0
```

Comment:
since the purpose of the dev container is to work on the actual source, shouldn't we maybe use a plain alpine image to avoid conflicts?

Reply:
Using a plain alpine or ubuntu image spends more time compiling the dependencies.
We have the `postCreateCommand` option which will be executed in the initial phase of the new container, it will overwrite Kong's LuaRocks packages. 
https://github.com/Kong/kong/blob/27d10142ce6f4167fc0da3cd4d6ca9d65768ecee/.devcontainer/devcontainer.json#L19

Gojira also use this method as an option to start a dev environment: `gojira up --image kong:2.7.0`, see: https://github.com/Kong/gojira/blob/master/docs/manual.md#using-kong-release-images-with-gojira

Other popular projects like dapr also use an existing docker image to start:
https://github.com/dapr/dapr/blob/10784ac6e5886fb4242eac73c4f13a8962d17043/.devcontainer/devcontainer.json#L4

Reply:
ye, I saw that. I'm just mildly concerned that stray artifacts have an impact on the sources. 

gojira, when used without the `--image` flag, it also uses a plain _ubuntu_ image. https://github.com/Kong/gojira/blob/master/docker/Dockerfile#L1

Either way, if you don't think my concerns are valid, I'll remain silent and approve this PR :)